public/static/js/version.js

```diff
@@ -73,7 +73,7 @@ mciModule.controller('VersionController', function($scope, $rootScope, $location
   }
 
   $scope.addSubscription = function() {
-    promise = addSubscriber($mdDialog, $scope.triggers);
+    promise = addSubscriber($mdDialog, $scope.triggers, {"jira-issue": true, "evergreen-webhook": true});
```

Comment:
You've got the constants for these fields, any reason you didn't use them?

Reply:
fixed